src/main/java/appeng/tile/storage/SkyChestTileEntity.java

```diff
@@ -58,16 +59,16 @@ public SkyChestTileEntity(TileEntityType<? extends SkyChestTileEntity> type) {
     @Override
     protected void writeToStream(final PacketBuffer data) throws IOException {
         super.writeToStream(data);
-        data.writeBoolean(this.getPlayerOpen() > 0);
+        data.writeBoolean(this.numPlayersUsing > 0);
     }
 
     @Override
     protected boolean readFromStream(final PacketBuffer data) throws IOException {
         final boolean c = super.readFromStream(data);
-        final int wasOpen = this.getPlayerOpen();
-        this.setPlayerOpen(data.readBoolean() ? 1 : 0);
+        final int wasOpen = this.numPlayersUsing;
+        this.numPlayersUsing = data.readBoolean() ? 1 : 0;
```

Comment:
As `numPlayersUsing` is now used client and serverside, the comment about it being server only should probably also be removed.